README.md

```diff
@@ -0,0 +1,18 @@
+## Silverstripe Non-Blocking File-Based Sessions
+
+[![CI](https://github.com/silverstripe/silverstripe-non-blocking-sessions/actions/workflows/ci.yml/badge.svg)](https://github.com/silverstripe/silverstripe-non-blocking-sessions/actions/workflows/ci.yml)
+
+## Overview
+
+Provides a non-blocking file-based session save handler for Silverstripe CMS 5.
+
+> [!WARNING]
+> This is not needed for Silverstripe CMS 6
```

Comment:
```suggestion
> [!INFO]
> This module is not needed for Silverstripe CMS 6+ because this functionality is built into `silverstripe/framework`
```

Reply:
Done (but added "from 6.1.0 onwards")